modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-2_{context}"]
+= Release notes for {gitops-title} 1.12.2
+
+{gitops-title} 1.12.2 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.2_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-12-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.12.2 security update advisory
+
+Issued: 2024-05-06
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-12-2_{context}"]
+== New features
+
+* With this update, support has been provided for the `must-gather` tool in the Argo Rollouts Operator. This update provides support for the following enhancements:
+
+** Logs for Argo Rollouts Operator Pods and Argo Rollouts Pods
+** Contents of Argo Rollouts Manager (CR) and Argo Rollouts (CR)
+** Contents of `Deployment`, `Statefulset`, `ConfigMaps` Custom Resources (CR) created by `must-gather` tool in the Argo Rollouts Operator
+link:https://issues.redhat.com/browse/GITOPS-3947[GITOPS-3947]
+
+[id="fixed-issues-1-12-2_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE, and EKS clusters as this binary was not available in the GitOps container. This update fixes the issue as the `argocd-k8s-auth` binary has been added in the GitOps container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
```

Comment:
```suggestion
* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE and EKS clusters because this binary was not available in the {gitops-shortname} container. This update fixes the issue by adding the `argocd-k8s-auth` binary in the {gitops-shortname} container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
```

Reply:
Fixed.